### DISCUSSION TITLE
NeMo Curator

### DISCUSSION THREAD

**Discussion by Joseph-Kassab** [DrAlzahraniProjects/csusb_fall2024_cse6550_team2#7] [Coding]
**Professor's Instructions:**

All students must contribute to all 24 topics, including tasks such as testing, improving documentation, reviewing code, enhancing performance, creating guides, troubleshooting, and assisting with deployment and integration.

**Guidelines:**

- Leave comments to write your contributions every week in the corresponding task discussion.

- Do not copy and paste large content (more than 10 lines), except for relevant error messages.

- Links can be shared.

- For error diagnosis, input errors into ChatGPT and share the link to the ChatGPT response for suggestions.

**Comment by Prasanth-Kandikatla** [comment]
Hi guys, just now gone through NVIDIA NeMo Curator. Got to know more about data downloading and text extraction. Interesting stuff below(link)

https://developer.nvidia.com/blog/curating-custom-datasets-for-llm-training-with-nvidia-nemo-curator/
